homeassistant/components/light/__init__.py

```diff
@@ -123,10 +123,7 @@
     ATTR_FLASH: vol.In([FLASH_SHORT, FLASH_LONG]),
 })
 
-LIGHT_TOGGLE_SCHEMA = vol.Schema({
-    ATTR_ENTITY_ID: cv.comp_entity_ids,
-    ATTR_TRANSITION: VALID_TRANSITION,
-})
+LIGHT_TOGGLE_SCHEMA = LIGHT_TURN_ON_SCHEMA
```

Comment:
On a second look, do all attributes of LIGHT_TURN_ON_SCHEMA make sense ? Shouldn't it be a subset ?

Reply:
Under the hood it calls turn_on, so I think everything that is accepted by turn_on could be passed there ( https://github.com/home-assistant/home-assistant/blob/519315f9c82ba9f0ea0bdcc622b4952b1c0a605c/homeassistant/helpers/entity.py#L470 ).